src/components/ItemPriceRange/ItemPriceRange.tsx

```diff
@@ -87,6 +90,14 @@ export function ItemPriceRange(props: Props) {
         }
     }
 
+    let onRangeChangeClick = (newRange) => {
+        trackEvent({
+            category: 'changeItemPriceRange',
+            action: newRange,
+        })
+        onRangeChange(newRange);
```

Comment:
execute the event handler first and only then track the click.
Fast UI is more important than tracking
```suggestion
    let onRangeChangeClick = (newRange) => {
        onRangeChange(newRange);
        trackEvent({
            category: 'changeItemPriceRange',
            action: newRange,
        })
```